crates/test_utils/tests/connectors/checkout_wh_ui.rs

```diff
@@ -18,8 +18,12 @@ async fn should_make_webhook(web_driver: WebDriver) -> Result<(), WebDriverError
         &format!("{CHEKOUT_BASE_URL}/saved/18"),
         vec![
             Event::Trigger(Trigger::Click(By::Id("card-submit-btn"))),
-            Event::Assert(Assert::IsPresent("status")),
-            Event::Assert(Assert::IsPresent("succeeded")),
+            Event::Trigger(Trigger::Sleep(8)),
+            Event::Assert(Assert::IsPresent("Google")),
+            Event::Assert(Assert::ContainsAny(
+                Selector::QueryParamStr,
+                vec!["status=succeeded", "status=processing"],
+            )),
         ],
         5,
         "succeeded",
```

Comment:
You need to change this too succeeded or processing . Because payment status we are asserting for both states .  Checkout sends webhooks both states